### PR TITLE
New server abstract class in common smo

### PR DESCRIPTION
**Pull request by KevinMicrosoft**

### CODE REVIEW

File: ossdbtoolsservice/language/completion_refresher.py [hunk 1]
```diff
@@ -27,7 +27,7 @@ class CompletionRefresher:
     def __init__(self, connection: ServerConnection, logger: Logger = None):
         self.connection = connection
         self.logger: Logger = logger
-        self.server: Server = None
```
Comment (swjain23) [inline ossdbtoolsservice/language/completion_refresher.py:30]: Why are we making it to PGServer? Isnt this in common class?

File: ossdbtoolsservice/edit_data/smo_edit_table_metadata_factory.py [hunk 2]
```diff
@@ -6,20 +6,20 @@
...
 from ossdbtoolsservice.metadata.contracts.object_metadata import ObjectMetadata
 from ossdbtoolsservice.query.contracts import DbColumn
+from pgsmo import PGServer
+from pgsmo.objects.table.table import Table  # noqa
+from pgsmo.objects.table_objects.column import Column  # noqa
 
 
 class SmoEditTableMetadataFactory:
 
     def get(self, connection: ServerConnection, schema_name: str, object_name: str, object_type: str) -> EditTableMetadata:
```
Comment (swjain23) [inline ossdbtoolsservice/edit_data/smo_edit_table_metadata_factory.py:21]: This looks like a common class between PG and Mysql, why are we chaning this to PGServer?

Comment (KevinMicrosoft) [reply ossdbtoolsservice/edit_data/smo_edit_table_metadata_factory.py:21]: This file originally used the pgsmo server, so I didn't want to change the functionality. But now that I'm looking at it, I think I should just go ahead and switch to the new server class!

File: pgsmo/objects/server/server.py [hunk 3]
```diff
@@ -7,6 +7,7 @@
 from urllib.parse import ParseResult, urlparse, quote_plus       # noqa
 
 from ossdbtoolsservice.driver import ServerConnection
```
Comment (swjain23) [inline pgsmo/objects/server/server.py:9]: Can we name this file pgserver? Generally class name should match the file name

File: mysqlsmo/objects/server/server.py [hunk 4]
```diff
@@ -7,6 +7,7 @@
 from urllib.parse import ParseResult, urlparse, quote_plus       # noqa
 
 from ossdbtoolsservice.driver import ServerConnection
+from smo.common.server import Server
```
Comment (swjain23) [inline mysqlsmo/objects/server/server.py:10]: Can we name this file mysqlserver?

File: mysqlsmo/__init__.py [hunk 5]
```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
```
Comment (swjain23) [inline mysqlsmo/__init__.py:3]: It looks good now, lets just change the name of the pg and MySQL server files to keep the naming consistent with the class name.

**Pull request closed without merging**